profiles/Jonas-setup.yaml

```diff
@@ -1,31 +1,41 @@
 ---
-safe_room: 4377 # bnb
-# Hunting Settings
-
+attunement_rooms: #Non-justice attunement rooms near my safe room 
```

Comment:
I'm not concerned about it, but I will remove the comment that its near my safe room